### PR TITLE
Adding saketkr24 coordinates

### PR DESCRIPTION
**Pull request by saketkr24**
location is added

### CODE REVIEW

**Comment by saketkr24** [comment]
Try again

**Comment by mrosenschein** [comment]
:alien:

File: pin/saketkr24hyd [hunk 1]
```diff
@@ -0,0 +1 @@
+echo "this is new file"
```
Comment (hectorsector) [inline pin/saketkr24hyd]: @saketkr24 you'll need to make sure the file has a `.json` extension, and that it your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

**Comment by beardofedu** [comment]
@saketkr24 thank you for checking out our [on-demand classes](https://services.github.com/on-demand/). Since this file is not valid per the examples in the course, I am going to go ahead and close this Pull Request. If you would like to complete the activity, please feel free to re-open this Pull Request or start again. If you have any questions, please let us know!

**Pull request closed without merging**